src/network/networkd-dhcp4.c

```diff
@@ -1449,6 +1461,96 @@ int config_parse_dhcp_request_options(
         return 0;
 }
 
+int config_parse_dhcp_send_options(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        Network *network = data;
+        const char *x;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+        assert(data);
+
+        if (isempty(rvalue)) {
+                network->dhcp_send_options = ordered_hashmap_free_with_destructor(network->dhcp_send_options, sd_dhcp_option_unref);
+                return 0;
+        }
+
+        r = ordered_hashmap_ensure_allocated(&network->dhcp_send_options, &trivial_hash_ops);
+        if (r < 0)
+                return log_oom();
+
+        for (x = rvalue;;) {
```

Comment:
Are you sure that multiple option assignments should be allowed in one line?
I think it'd be simpler to always require just one option setting per line.

Reply:
Was not the previous code doing that ? Now I am confused .

Reply:
Of course you only want to consider the first one. The question is whether to allow one word on the rhs or multiple words.